### PR TITLE
LPD-22018 - Hide info alert when scriptManagementConfigurationDefined is true

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-22018

### CODE REVIEW

File: modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx [hunk 1]
```diff
@@ -76,6 +78,10 @@ export default function ScriptManagementContainer({
 					: 'danger',
 			});
 
+			if (editScriptManagementConfigurationResponse.ok) {
+				setTimeout(() => window.location.reload(), 1000);
+			}
```
Comment (igor-franca) [inline modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx:83]: Hey @danvb112, could we use a state to validate the alert, thus avoiding the need to reload the page?

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @danvb112, just a detail.

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx:90]: Done! Thanks @igor-franca

**Comment by danvb112** [comment]
ci:test:sf

**Review by igor-franca — APPROVED** [review]
Great work Daniel, everything seems to work as expected


https://github.com/liferay-objects/liferay-portal/assets/61854510/38cc21bf-0f7f-4687-9cf9-e0a6eb218197

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward:force

**Pull request closed without merging**